File: 2-Basic-Mathematical-Knowledge-for-Algorithms/001-Print 5+N/javascript/main.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

// 1. 入力処理
rl.on('line', (line) => {
    // 文字列として受け取るので、parseIntで整数に変換
    const n = parseInt(line, 10);

    // 2. 計算処理
    const sum = 5 + n;

    // 3. 出力処理
    console.log(sum);

    // 入力処理を終了
    rl.close();
});

File: 2-Basic-Mathematical-Knowledge-for-Algorithms/001-Print 5+N/rust/src/main.rs
use std::io;

fn main() {
    // 1. 入力処理
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    // 読み取った文字列から改行コードを取り除き、整数に変換
    let n: i32 = input.trim().parse().unwrap();

    // 2. 計算処理
    let sum = 5 + n;

    // 3. 出力処理
    println!("{}", sum);
}
